boards/arm/nucleo_l4r5zi/Kconfig.defconfig

```diff
@@ -9,7 +9,7 @@ config BOARD
 	default "nucleo_l4r5zi"
 
 choice STM32_LPTIM_CLOCK
-	default STM32_LPTIM_CLOCK_LSI
+	default STM32_LPTIM_CLOCK_LSE
 	depends on STM32_LPTIM_TIMER
 endchoice
```

Comment:
I think we could completely remove this choice section.

Reply:
yes, removed

Reply:
yes, removed